graveler/pgiterator.go

```diff
@@ -109,10 +123,12 @@ func (ri *PGRepositoryIterator) SeekGE(id RepositoryID) bool {
 	} else {
 		ri.buf = make([]*RepositoryRecord, 0)
 	}
-	return true
 }
 
 func (ri *PGRepositoryIterator) Value() *RepositoryRecord {
+	if ri.postInit || ri.postSeekGE {
```

Comment:
It would be nice to also return nil in case of error

Reply:
Done.